### PR TITLE
Split client and server readme

### PR DESCRIPTION
**Pull request by AleksSenkou**
**Changes:**
* [x] Separate client's and server's bin/setup and bin/update scripts
* [x] Update client README with actual information about available scripts
* [x] Add server README with tips, links and advices

### CODE REVIEW

**Comment by AleksSenkou** [comment]
[When I read project's documentation, I'd like separately see difference in client and server docs, so it will be easier for me to understand](https://trello.com/c/TW3hidFx/41-when-i-read-projects-documentation-id-like-separately-see-difference-in-client-and-server-docs-so-it-will-be-easier-for-me-to-un)

File: server/README.md [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+
+* `yarn dev` - runs dev server
+* `yarn dev:test` - runs dev server in testing env
+* `yarn dev:kill` - kills dev server in testing env
+* `yarn debug` - run dev server in debug mode
+
+Testing:
+
+* `yarn test` - run all tests
+* `yarn test:api` - run endpoints dredd tests and kills server process
```
Comment (DDKatch) [inline server/README.md]: @AleksSenkou does it kill only test server?

File: server/bin/setup [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+
+echo "Runing migrations:"
+node_modules/.bin/sequelize db:migrate
+
+echo "Runing migrations for test db:"
+NODE_ENV=test node_modules/.bin/sequelize db:migrate
+
+# GLOBAL modules
+echo "For interacting with sequalize cli directly from command line:"
+echo "NOTE: you can skip this step and use node_modules/.bin/sequelize 😉"
```
Comment (DDKatch) [inline server/bin/setup]: @AleksSenkou how is it possible to skip this step without editing this file?

**Review by DDKatch — APPROVED** [review]
Looks Good To Merge

File: server/bin/setup (hunk 2 above)
Comment (AleksSenkou) [reply server/bin/setup]: @DDKatch just to remove lines from the file. It's oriented on advanced devs. For beginners `bin/setup` will be enough, thoughts?

File: server/README.md (hunk 1 above)
Comment (AleksSenkou) [reply server/README.md]: Yeah, see: https://github.com/datarockets/loopback-to-do/blob/master/server/package.json#L17

Comment (AleksSenkou) [reply server/README.md]: Do you think it should be described? like `and kills test server process`

**Comment by nikitasakov** [comment]
LGTM

File: server/README.md (hunk 1 above)
Comment (AleksSenkou) [reply server/README.md]: Fixed

**Pull request merged**